internal/fs/fs.go

```diff
@@ -2083,17 +2084,15 @@ func (fs *fileSystem) WriteFile(
 	// Serve the request.
 	if fs.isWriteBufferEnabled(in) {
```

Comment:
Should we add a TODO that this logic requires unit tests (we can then look at all TODOs before code shipping that all the important TODOs are taken care of or not)

Reply:
added the TODO, thanks